.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,3 +18,13 @@ If this pull request addresses a Help Wanted ticket, please link the relevant Gi
 
 Otherwise, link the JIRA ticket.
 -->
+
+
+#### Release Note
+<!--
+If no, just write "NONE" in the release-note block below.
+If yes, a release note is required:
+Enter your past-tense release note in the block below.
+-->
```

Comment:
Are there any additional guidelines on how the message should be written? E.g. all bug fixes could be `Fixed ...`. Are they allowed to be funny?

Reply:
since we have permission to edit the PR description we can always review and make editions.
if the contributor is unsure if need release notes, they can ask for a maintainer.

but agree, we should document that in the developer documentation